pkg/portworx/pxVolumeOps.go

```diff
@@ -503,3 +504,50 @@ func (p *pxVolumeOps) GetPxPvcs() ([]*kubernetes.PxPvc, error) {
 	}
 	return p.pxPvcs, nil
 }
+
+func addToNodeIds(nodeIds map[string]bool, nodes []string) {
```

Comment:
These two functions will also need unit tests.

Reply:
Have added UT to test GetAllNodesForVolume. This will ensure getMidsArray and addToNodeIds are tested.